.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
pre-commit linter is pain. After a few commits you'll skip it. It's better move linter in pre-push before tests